homeassistant/components/alarmdecoder/__init__.py

```diff
@@ -205,24 +120,54 @@ def handle_rel_message(sender, message):
     controller.on_close += handle_closed_connection
     controller.on_expander_message += handle_rel_message
 
-    hass.data[DATA_AD] = controller
+    hass.data[DOMAIN] = controller
```

Comment:
We usually store the data under `hass.data[DOMAIN][entry.entry_id] = controller` so we can allow multiple instances.  Its possible someone will have two systems.

Reply:
done in https://github.com/home-assistant/core/pull/37998/commits/42a8cff068c03041b38c1fd31a2936203c3ab67c#diff-0084eacb87a2f8212f5bd9e29ca3282bR128-R132 :+1: